web/packages/selfhosted/test/polyfill/percent-height-when-zero/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>PERCENT-BASED-HEIGHT</title>
+</head>
+<body>
+  <div style="height: 0px;">
+    <div>
+      <embed id="content_1" width="100%" height="100%" src="https://ruffle-regressions.github.io/logo-anim.swf"></embed>
+    </div>
+  </div>
+  <div style="height: 0px;">
+    <embed id="content_2" width="100%" height="100%" src="https://ruffle-regressions.github.io/logo-anim.swf"></embed>
+  </div>
+  <div style="height: 50px;">
+    <embed id="content_3" width="100%" height="100%" src="https://ruffle-regressions.github.io/logo-anim.swf"></embed>
+  </div>
+  <embed id="content_4" width="100%" height="100%" src="https://ruffle-regressions.github.io/logo-anim.swf"></embed>
+  <embed id="content_5" width="100%" height="50px" src="https://ruffle-regressions.github.io/logo-anim.swf"></embed>
+  <embed id="content_6" width="100%" height="0px" src="https://ruffle-regressions.github.io/logo-anim.swf"></embed>
+  <div id="result"><div id="result_1"></div><div id="result_2"></div><div id="result_3"></div><div id="result_4"></div><div id="result_5"></div></div>
+  <script>
+  function writeResult(res, index) {
+    const resultDiv = index < 6 ? document.getElementById("result_" + index) : document.createElement("div");
+    resultDiv.textContent = res;
+    resultDiv.id = "result_" + index;
+    const result = document.getElementById("result");
+    if (index === 6) result.appendChild(resultDiv);
+  }
+  function checkClientHeight(content, index) {
+      writeResult(content.clientHeight, index);
+  }
+  document.addEventListener("DOMContentLoaded", () => {
+      for (let i = 1; i <= 6; i++) {
+        setTimeout(() => checkClientHeight(document.getElementById("content_" + i), i), 15000);
```

Comment:
What should be done here instead of a 15 second timeout? An 8 second timeout was not long enough.